models/dex/chains/mev/sandwiches/dex_arbitrum_sandwiched.sql

```diff
@@ -0,0 +1,19 @@
+{% set blockchain = 'arbitrum' %}
+
+{{ config(
+        
+        schema = 'dex_' + blockchain,
+        alias = 'sandwiched',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['blockchain','tx_hash', 'evt_index']
+)
```

Comment:
you can add this back in:
`incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]`

the difference here is that you're targeting the filter, so you'll need target column name. i've udpated it above to `block_time` as it should be